rke/templates/templates.go

```diff
@@ -279,6 +284,7 @@ func getTemplates() map[string]string {
 
 		aciv500: AciTemplateV500,
 		aciv523: AciTemplateV523,
+		aciv5233: AciTemplateV5233,
```

Comment:
Nit, but we want variables and values to align, see other blocks like this.

Reply:
addressed